modules/custom/az_news/config/install/views.view.az_news.yml

```diff
@@ -581,31 +582,174 @@ display:
         - 'user.node_grants:view'
         - user.permissions
       tags: {  }
-  az_paged_row:
+  az_news_category_pages_view:
+    id: az_news_category_pages_view
+    display_title: 'AZ News Category Pages'
     display_plugin: page
+    position: 9
+    display_options:
+      title: ''
+      empty:
+        title:
+          id: title
+          table: views
+          field: title
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: title
+          empty: true
+          title: ''
+      arguments:
+        name:
+          id: name
+          table: taxonomy_term_field_data
+          field: name
+          relationship: term_node_tid
+          group_type: group
+          admin_label: ''
+          entity_type: taxonomy_term
+          entity_field: name
+          plugin_id: string
+          default_action: default
+          exception:
+            value: all
+            title_enable: false
+            title: All
+          title_enable: true
+          title: '{{ path(''<current>'')|split(''/'')|last|replace({''-'': '' ''})|capitalize }}'
```

Comment:
This twig syntax seems to work fine but might fail if the taxonomy term itself includes a hyphen.

Consider using this module:
https://www.drupal.org/project/views_taxonomy_term_name_into_id

Possibly also including this patch for the module:
https://www.drupal.org/project/views_taxonomy_term_name_into_id/issues/3092498

Reply:
Sweet, I'll redo this the way CG recommended, I'll look into this module. Thanks!

Reply:
@joshuasosa nice find with that module and patch, good point about hyphenated term names. @Krystation that is some fancy twig